src/pages/plantDetailPage/PlantDetailPage.tsx

```diff
@@ -3,103 +3,120 @@ import NavItem from "../../components/navItem/NavItem.tsx";
 import PlantInfo from "../../components/plantDetails/PlantInfo.tsx";
 import { Outlet, Route, Routes, useParams } from "react-router-dom";
 import Text from "../../components/text/Text.tsx";
-import { FormValues } from "../../components/form/Form.types.ts";
 import {
   PlantNavItemStyled,
   PlantNavStyled,
 } from "../../components/plantDetails/PlantDetail.styles.ts";
+import { pb, PLANTS_COLLECTION } from "../../Backend.constants.ts";
+import { useQuery } from "@tanstack/react-query";
 
 const PlantDetailPage = () => {
-  const plantsCollection: FormValues[] =
-    JSON.parse(localStorage.getItem("plantsList") || "") || [];
-
   const { plantId } = useParams();
 
-  const currentPlantId = plantsCollection.find(
-    (singlePlant) => singlePlant.id === plantId,
-  );
+  const getPlantsList = async () => {
```

Comment:
it's duplicated, but more importantly you shouldn't need to fetch the entire list here. Is there another method in pb that let's you extract only one item from a collection? That's the only thing you need here?

This is important for caching reasons and performance in the long run. 
If you fetch only one element via react-query and save it as a single entry in cache then when you modify/delete (mutate in general) that item you don't need to refetch the entire list of plants again. You will only tell the cache that this one plant has e.g. different watering requirements.